guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -24,12 +22,6 @@ endif::[]
 
 .File System Guidelines
 
-* Use the XFS file system for {ProjectNameX} because it does not have the inode limitations that `ext4` does.
-Because {ProductName} uses a lot of symbolic links it is likely that your system might run out of inodes if using `ext4` and the default number of inodes.
-
-* Do not use NFS for Pulp data storage.
-Using NFS for Pulp has a negative performance impact on content synchronization.
```

Comment:
Another question for the pulp folks, since we're not dealing with lots of symlinks, i'd think the ban on nfs could go away, yes it might be somewhat slower, but the extra high latency wouldn't be nearly as much of a big deal (compared to creating ~20K symlinks).

Reply:
I agree.